tests/benchdnn/dnn_types.cpp

```diff
@@ -973,6 +973,8 @@ std::ostream &operator<<(std::ostream &s, const attr_t::dropout_t &drop) {
     s << drop.p;
     if ((drop.seed != 0) || (drop.tag != tag::any)) s << ":" << drop.seed;
     if (drop.tag != tag::any) s << ":" << drop.tag;
+    if (drop.offset != 0) s << ":" << drop.offset;
+    if (drop.use_host_scalars) s << ":" << drop.use_host_scalars;
```

Comment:
This is misaligned with the parser which expects arguments ordered. Tag should be printed if offset and scalars are set and offset should be printed if scalars are used.